src/main/java/io/github/fusionflux/portalcubed/framework/extension/EntityExt.java

```diff
@@ -0,0 +1,12 @@
+package io.github.fusionflux.portalcubed.framework.extension;
+
+public interface EntityExt {
+	int DISINTEGRATE_TICKS = 3 * 20;
```

Comment:
I don't like storing these here

Reply:
Would it be better if EntityExt was renamed to DisintegrationEntityExtensions or something like that

Reply:
no. I guess it's fine